### PR TITLE
[CI:DOCS] Update release notes for v2.2.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -38,6 +41,10 @@
...
 - The `podman save` command now strips signatures from images it is exporting, as the formats we export to do not support signatures ([#7659](https://github.com/containers/podman/issues/7659)).
 - A new `Degraded` state has been added to pods. Pods that have some, but not all, of their containers running are now considered to be `Degraded` instead of `Running`.
+- Podman will now print a warning when conflicting network options related to port forwarding (e.g. `--publish` and `--net=host`) are specified when creating a container.
+- The `--restart on-failure` and `--rm` options for containers no longer conflict. When both are specified, the container will be restarted if it exits with a non-zero error code, and removed if it exits cleanly ([#7906](https://github.com/containers/podman/issues/7906)).
+- Remove Podman will no longer use settings from the client's `containers.conf`; defaults will instead be provided by the server's `containers.conf` ([#7657](https://github.com/containers/podman/issues/7657)).
```
Comment (Luap99) [inline RELEASE_NOTES.md:46]: typo `Remote Podman`

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -3,12 +3,14 @@
...
+- The `podman run` command now supports the `--network-alias` option to set network aliases (additional names the container can be accessed at from other containers via DNS if the `dnsname` CNI plugin is in use). Aliases can also be added and removed using the new `podman network connect` and `podman network disconnect` commands. Please note that this requires a new release (v1.1.0) of the `dnsname` plugin, and will only work on newly-created CNI networks.
 - The `podman generate kube` command now features support for exporting container's memory and CPU limits ([#7855](https://github.com/containers/podman/issues/7855)).
 - The `podman play kube` command now features support for setting CPU and Memory limits for containers ([#7742](https://github.com/containers/podman/issues/7742)).
+- The `podman play kube` command now supports persistent volumes claims using Podman named volumes.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:10]: claims?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:46]: Not sure what "Remove Podman" is.  "Remote" perhaps?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -103,6 +110,14 @@
...
 - Fixed a bug where the `--tls-verify` option of the `podman play kube` command had its logic inverted (`false` would enforce the use of TLS, `true` would disable it).
+- Fixed a bug where the `podman network rm` command would error when trying to remove `macvlan` networks and rootless CNI networks ([#8491](https://github.com/containers/podman/issues/8491)).
+- Fixed a bug where Podman was not setting sane defaults for missing `XDG_` environment variables.
+- Fixed a bug where remote Podman would check if volume paths to be mounted in the container existed on the host, not the server ([#8473](https://github.com/containers/podman/issues/8473)).
+- Fixed a bug where the `podman manifest create` and `podman manifest add` commands on local images would drop any images in the manifest not pulled by the host.
+- Fixed a bug where networks made by `podman network create` did not include the `tuning` plugin, and as such did not support setting custom MAC addresses ([#8385](https://github.com/containers/podman/issues/8385)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:117]: I think "tuning" is correct, but just double checking.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -103,6 +110,14 @@
...
+- Fixed a bug where container healthchecks did not use `$PATH` when searching for the Podman executable to run the healthcheck.
+- Fixed a bug where the `--ip-range` flag to `podman network create` did not proper handle non-classful subnets when calculating the last usable IP for DHCP assignment ([#8448](https://github.com/containers/podman/issues/8448)).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:119]:
```suggestion
- Fixed a bug where the `--ip-range` flag to `podman network create` did not properly handle non-classful subnets when calculating the last usable IP for DHCP assignment ([#8448](https://github.com/containers/podman/issues/8448)).
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -112,22 +127,28 @@
...
+- Fixed a bug where the Ping endpoint sent an extra newline at the end of its response where Docker did not.
 - Fixed a bug where the Compat Logs endpoint for containers did not send a newline character after each log line.
 - Fixed a bug where the Compat Logs endpoint for containers would mangle line endings to change newline characters to add a preceding carriage return ([#7942](https://github.com/containers/podman/issues/7942)).
 - Fixed a bug where the Compat Inspect endpoint for Containers did not properly list the container's stop signal ([#7917](https://github.com/containers/podman/issues/7917)).
 - Fixed a bug where the Compat Inspect endpoint for Containers formatted the container's create time incorrectly ([#7860](https://github.com/containers/podman/issues/7860)).
-- Fixed a bug where the Compat Inspect endpoint for Containers did not include complete network information on the container.
+- Fixed a bug where the Compat Inspect endpoint for Containers did not include the container's Path, Args, and Restart Count.
+- Fixed a bug where the Compat Inspect endpoint for Containers prefixed added and dropped capabilities with `CAP_` (Docker did not do so).
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:137]: soft suggestion
```suggestion
- Fixed a bug where the Compat Inspect endpoint for Containers prefixed added and dropped capabilities with `CAP_` (Docker does not currently do so).
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
Done picking.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (mheon) [reply RELEASE_NOTES.md:10]: Yes. https://kubernetes.io/docs/concepts/storage/persistent-volumes/ - we are supporting PVCs (Persistent Volume Claims).

File: RELEASE_NOTES.md (hunk 3 above)
Comment (mheon) [reply RELEASE_NOTES.md:117]: It is.

**Comment by mheon** [comment]
Repushed with fixed
@containers/podman-maintainers PTAL

**Review by QiWang19 — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**